### PR TITLE
Create Week 1

### PR DESCRIPTION
**Pull request by albanpuech**
I created the file for the reports of the first week and I added my report.

### CODE REVIEW

**Comment by ngthanhvinh** [comment]
@albanpuech It's great that you have written the report. Your work during the week is really cool! However, personally I think it would be better to write the report on Google Docs and push it on Trello. Github is supposed to store code only.

Regarding the data in the backtest environment, you actually do not need to read the input. You only have to fill in the start/ end dates, and the code should load the data itself. Overall, you just need to implement the `MainStrategy` class (see the `mainstrategy.cpp` and `strategiesheader.hpp` files for more details).

**Comment by albanpuech** [comment]
@ngthanhvinh Hey, if I understood well what is written, we have to create a folder on our repository to store the reports and add one file per week in which we all write our report.

For the data (input) I was talking about my strategy, not about the backtest environment.

I have to look to the classes already existing to see what I have to do. The classes I want to create next week are specific to my strategy, but I probably didn't understand something correctly. 

Thank you !

Cheers,

File: reports/Week 1 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- I did a report and a description of the algorithm that I have to implement for my strategy. (available on Trello)
+- I tried to make the backtest environment that we will use to check our implementation work on my computer running windows but I could not do it.
+- I installed Ubuntu on a virtual machine to finally run it.
+
+What I have to do :
+- Start to implement my algorithm for the momentum strategy using the CSV file we already have as input for test. finishing to create all the classes needed to compute the momentum
+at any date would be great.
+- Think about the way to translate the numerical result into a binary recommendation : buy/ sell.
+
+
```
Comment (ghost) [inline reports/Week 1:17]: Hello Alban,
Good you created this so we can have all of our individual weekly reports in a single file as recommended.

**Review by ghost — APPROVED** [review]
Hello Alban,
Good you created this so we can all have our individual weekly reports in a single file.

**Review by ngthanhvinh — APPROVED** [review]

**Pull request merged**